src/styles/report.scss

```diff
@@ -0,0 +1,21 @@
+
+
+.reportSubmit{
```

Comment:
@nyabongoedgar , please remove this !important from your styles there are better ways to do this.

Reply:
Thank you. Issue has been solved.